applications/produce_array_config.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application is a simple example of how to produce sim_telarray config \
+    files for a given array.
+
+    All the input required has to be given as a yaml file by the command \
+    line argument array_config.
+
+    The required entries in the array_config file are:
+
+    'site': South or North
+
+    'layoutName': name of a valid layout array.
+
+    'modelVersion': name of a valid model version.
+
+    'default': telescope model names to be assigned to each telescope size by default. \
+    It must contain entries for 'LST' and 'MST' (and 'SST' in case of South site).
+
+    As optional data, specific telescope models can be assigned to individual telescopes. \
+    This is done by the entries with the name of the telescope (as used by the layout \
+    definition, ex. L-01, M-05, S-10).
+
+    Each telescope model can be set in two ways.
+
+    a) A single str with the name of telescope model.
+    Ex. 'M-05': 'NectarCan-D'
```

Comment:
Should be 'NectarCam-D'? What does does the 'D' stand for?

Reply:
Fixed!

D stand for "design". This comes from the convention we came up with for the naming of the telescope models.
Real telescopes will have single numbers (e.g. LST-1) and design models should preferably contain a D (e.g LST-D234).

Reply:
OK - forgot about the 'D' for design. All good!